recipes/libdxfrw/all/conanfile.py

```diff
@@ -16,44 +20,43 @@ class LibdxfrwConan(ConanFile):
         "shared": False,
         "fPIC": True
     }
-    generators = "cmake"
-    homepage = "https://github.com/LibreCAD/libdxfrw"
+    _cmake = None
 
-    def _get_libdxfrw_folder(self):
-        return "libdxfrw-" + self.version
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
-        tools.replace_in_file(self._get_libdxfrw_folder() + "/CMakeLists.txt", "set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)",
-                              '''set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()''')
+        extracted_folder = self.name + "-" + self.version
+        os.rename(extracted_folder, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if not self._cmake:
+            self._cmake = CMake(self)
+            self._cmake.configure()
+        return self._cmake
 
     def build(self):
         cmake = self._configure_cmake()
         cmake.build()
 
-    def _configure_cmake(self):
-        cmake = CMake(self)
-        cmake.definitions["CMAKE_INSTALL_PREFIX"] = "install"
-        print(self.source_folder)
-        cmake.configure(source_folder=self._get_libdxfrw_folder())
-        return cmake
-
-    def package_info(self):
-        self.cpp_info.libs = ["dxfrw"]
-
-    def config_options(self):
-        if self.settings.os == "Windows":
-            del self.options.fPIC
-
     def package(self):
+        self.copy("COPYING", src=self._source_subfolder, dst="licenses")
         cmake = self._configure_cmake()
         cmake.install()
-        self.copy("*", src="install/bin", dst="bin")
-        self.copy("*",
-                  src="install/lib",
-                  dst="lib",
-                  excludes=('cmake', 'pkgconfig'))
-        self.copy("*", src="install/include", dst="include")
-        self.copy("COPYING", src=self._get_libdxfrw_folder(), dst="licenses")
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+
+    def package_info(self):
+        bin_path = os.path.join(self.package_folder, "bin")
+        self.output.info("Appending PATH env var: {}".format(bin_path))
```

Comment:
what binary is exported to the bin folder?

Reply:
It's `dxf2dxf`
https://github.com/LibreCAD/libdxfrw/tree/master/dwg2dxf

Reply:
thanks!